code/src/App.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from 'react';
+import { Provider} from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import labyrinth from 'reducers/labyrinth';
+
+import Labyrinth from 'components/Labyrinth';
+
+const reducer = combineReducers({
+	labyrinth: labyrinth.reducer
+});
+
+const store = configureStore({ reducer })
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
+	return (
+		<Provider store={store}>
+			<Labyrinth />
+		</Provider>
```

Comment:
Super nice and easy-to-grasp App file - like that the game is all inside the Labyrinth component.